src/ivoc/nrnmain.cpp

```diff
@@ -9,6 +9,7 @@ extern "C" {
 	int nrn_isdouble(double*, double, double);
 	int ivocmain(int, const char**, const char**);
 	extern int nrn_main_launch;
+	extern int nrn_noauto_dlopen_nrnmech;
```

Comment:
The reason I avoided nrn_auto_dlopen_nrnmech was because it would need a default value of 1 and it seemed safest to rely on a default value of 0 which then gets changed very early after launch